examples/oneview_ethernet_network.yml

```diff
@@ -66,7 +77,7 @@
         data:
           name: "{{ network_name }}"
           scopeUris: []
-      when: contents.api_version >= 300 and contents.api_version <= 500
+      when: currentVersion >= '300' and currentVersion <= '500'
```

Comment:
why api version interpreted as 'string'?

Reply:
we will get currentVersion in string format as it is coming from ansible_facts